File: yujeong/TwoSum.java
package yujeong;

public class TwoSum {
    public int[] twoSum(int[] nums, int target) {
        int [] result=new int[2];

        for(int i=0; i<nums.length; i++){ //배열의 길이만큼 for문 돌림
            for(int j=i+1; j<nums.length; j++){
                if(nums[i]+nums[j]==target){
                    result[0]=i;
                    result[1]=j;

                    return result;
                }
            }
        }
        return result;
    }
}

File: yeaji/twosum.go
func twoSum(nums []int, target int) []int {
    i := 0
    for i < len(nums)-1{
        j := i+1
        for j <= len(nums)-1 {
            if nums[i] + nums[j] == target {
                return []int{i, j}
            }
            j++
        }
        i++
    }
    return []int{}
}
